File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.js
/**
 * @param {number[]} nums
 * @return {number}
 */
let maximumCount = function (nums) {
    let p = 0, n = 0;

    for (num of nums) {
        if (num > 0) {
            p++;
        }
        else if (num < 0) {
            n++;
        }
    }

    return Math.max(p, n);
};

File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.py
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        n = len(nums)
        p = 0
        n = 0

        for num in nums:
            if num < 0:
                p += 1
            elif num > 0:
                n += 1

        return max(p, n)
